4_Methodology/toolbox.adoc

```diff
@@ -0,0 +1,63 @@
+ 
+= Tooblox for testing compliance to cPP for Mobile biometric enrolment and verification 
+:showtitle:
+:toc:
+:sectnums:
+:imagesdir: images
+:revnumber: 0.2
+:revdate: 2019-01-24
+
+== Introduction
+This document an its child documents contains a toolbox of Presentation Attack Species (PAS) for various biometric modalities. This toolbox shall be used to test the PAD functionality of a TOE pf a TOE during an evaluation in compliance to <<cPP>>.
+
+This toolbox document shall be used to test for the implementation of the SFR FIA_MBV_EXT.3 from <<cPP>>
+It contains content for the following biometric modalities 
+
+* Iris
+* Face
+* Fingerprint
+
+It should be noted that the toolbox as contained in this document focusses on a functional approach of testing as required by the assurance component ATE_IND.1_. Each toolbox contains the state of the art of the Presentation Attack Instruments of its respective modality. A TOE compliant to <<cPP>> is expected to reliably recognize these PAI. The functional approach of the toolboxes is however not suitable for any test in the context of AVA_VAN. 
+
+The following chapter introduces the overall test approach
+
+== Overall test approach
+Then testing approach bases on the following assumptions and proinciples
+
+* As it could not be assumed that every TOE will return a dedicated result for the PAD functionality, success for an presentation attack is defined for the whole biometric functionality, including the matcher. This means that - in order to succesfully overcome the TOE by the use of an artefact - a genuine person (the tester) has to be enrolled into the TOE, an artefact has to be created for the corresponding biometric modality of the tester and the artefact has to produce a match (i.e. a succesfull verification)
+* more?
+
+=== Preparational phase
+Before the actual test can start, the following pre-requisites need to be met:
+* It has to be ensured that the tester whose biometric data is used to produce the artefacts for testing is enrolled into the TOE. Sucesfull enrollment needs to be documented and verified by performing at least 5 genuine transactions with the TOE. No errors shall occur during the 5 genuine transactions. If any errors occur, the enrollment process shall be repeated. In case of repeated errors, it should be considered to use a different biometric characteristic (this could e.e. mean to use a different finger of a tester or even go so far to exchange the tester)
```

Comment:
I would recommend not specifying that it is the "tester" whose biometric data is used. Rather we should use the term "subject" to allow for the possibility that other folks can be used in this testing. I indicated changes below for you using italics - the italics are not intended for the final text.

"* It has to be ensured that the _subject_ whose biometric data is used to produce the artefacts for testing is enrolled into the TOE. _Successful_ enrollment needs to be documented and verified by performing at least 5 genuine transactions with the TOE. No errors shall occur during the 5 genuine transactions. If any errors occur, the enrollment process shall be repeated. In case of repeated errors, it should be considered to use a different biometric characteristic (this could e.e. mean to use a different finger of a _subject or even use a different subject__)"

Reply:
edited in #153